internal/forger/workflows_test.go

```diff
@@ -0,0 +1,30 @@
+package forger
+
+import (
+	"context"
+	"testing"
+
+	"github.com/gkampitakis/go-snaps/snaps"
+	"github.com/stretchr/testify/require"
+)
+
+func TestWorkflows(t *testing.T) {
+	p := Plan{
+		Name:    "hello-world",
+		Image:   "docker.io/busybox:latest",
+		Command: []string{"echo", "Hello, World!"},
+		Env: map[string]string{
+			"GREETING": "Hello",
+		},
+	}
+
+	ctx := context.Background()
+
+	tf, err := GetForgerForTarget("argo-workflows")
+	require.NoError(t, err)
+	require.IsType(t, ArgoWorkflowsForger{}, tf)
+
+	got, err := tf.Forge(ctx, p)
+	require.NoError(t, err)
+	snaps.MatchYAML(t, got)
```

Comment:
If you run this test locally, `make test`, changes will be done under the `internal/forger/__snapshots__` directory. Can you include those in the PR?

Reply:
I'll do so as a follow up PR.

Reply:
https://github.com/lcarva/smithron/commit/a0ef7bdada22b1025c13214a99aafc46ebbeebf9

Reply:
Thanks!